### PR TITLE
Move non-`network.*` attributes out of network.yaml

### PR DESCRIPTION
**Pull request by trask**
I missed this in #256

### CODE REVIEW

**Review by Oberon00 — APPROVED** [review]

**Review by AlexanderWert — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]
Change log entry maybe?

File: CHANGELOG.md
Comment (Oberon00) [inline CHANGELOG.md:63]: @joaopgrassi Why did you request this changelog entry? This should be 100% transparent to users and and just an internal detail. If we want want to have every PR in the changelog we should just auto-generate it. I would remove this again.

Comment (joaopgrassi) [reply CHANGELOG.md:63]: It was more of a question, not a request per se. I thought about the case where people might look for for them in the yaml and not finding and maybe then looking for the reason of why. Having it in the changelog could help discovering things.

Comment (trask) [reply CHANGELOG.md:63]: I personally didn't think it was necessary, but IMO it's up to @open-telemetry/specs-semconv-maintainers to decide where they want to draw the line when it comes to adding change log entries.

Comment (joaopgrassi) [reply CHANGELOG.md:63]: Sorry for the confusion here folks, I should have indicated in my comment why I thought a changelog would come in handy. I think it makes sense for the case I stated (mostly discovering things) but also OK with me without it.

**Comment by trask** [comment]
@open-telemetry/specs-semconv-maintainers I think this is good to merge

**Pull request merged**
